data/languages/mips.sinc

```diff
@@ -368,8 +368,8 @@ define pcodeop syscall;
     }
 
     # 0011 1100 000t tttt iiii iiii iiii iiii
-    :lui RT, immed                  		is prime=15 & rs=0 & RT & immed {
-        tmp:4 = immed << 16;
```

Comment:
I think this one was correct. `lui` uses the unsigned immediate 16 bit value, shifts left 16 bits and sign extends it.

Reply:
Somehow it got broken, I have no idea how. Maybe it's my project?

Reply:
yeah you were right